src/V3LinkDot.cpp

```diff
@@ -2683,6 +2683,21 @@ class LinkDotResolveVisitor final : public VNVisitor {
         m_inSens = true;
         iterateChildren(nodep);
     }
+    void visit(AstPatMember* nodep) override {
+        if (nodep->nodep()) return; // only do this mapping once
+        // If we have a TEXT token as our key, lookup if it's a LPARAM
+        if(AstText* textp = VN_CAST(nodep->keyp(), Text)){
+            UINFO(9, "visit " << nodep << endl);
+            // Lookup
+            VSymEnt* const foundp = m_curSymp->findIdFallback(textp->text());
+            if (foundp) {
```

Comment:
```suggestion
            if (AstVar* const varp = VN_CAST(foundp, Var)) {
```

Reply:
Casting directly from a VSymEnt* wasn't working, still needed to pull out the `nodep()`. Have wrapped foundp in an if, so that if we don't have a localparam to match (typo, etc) we aren't trying to take that nodep() from a nullptr.